CHANGES.md

```diff
@@ -1,3 +1,32 @@
+## edge-19.8.6
+
+* CLI
+  * Fixed missing `clusterDomain` in render RenderTapOutputProfile
+  * Added LINKERD2_PROXY_DESTINATION_GET_SUFFIXES env variable (thanks @arminbuerkle!)
+  * Merged the CLI 'installValues' type with Helm 'Values' type 
+    * Renamed template-values.go
+    * Defined new constructor of charts.Values type
+    * Moved all Helm values related code to the pkg/charts package
+    * Bumped dependency
+    * Used '/' in filepath to remain compatible with VFS requirement
+    * Added unit test to verify Helm YAML output
+    * Added unit test for Helm YAML validation (HA)
+* Controller 
+  * Added requirement for go 1.12.9 for controller builds
+  * Reorganized authority-parsing code so that the is-this-a-kubernete-service-name decision is decoupled from parsing
+    of authorities into their consituent parts
+* Web UI
+  * Added a "Linkerd Namespace" Grafana dashboard  
```

Comment:
This feature was thanks to `@bourquep` 😄 

I would add a few words about why this was useful, maybe:

Added a “Linkerd Namespace” Grafana dashboard, allowing users to view historical data for a given namespace, similar to CLI output for `linkerd stat deploy -n myNs`

Reply:
Great additional context 👍